angular/projects/admin-nrpti/src/app/mines/mines-detail/mines-detail.component.html

```diff
@@ -1,65 +1,96 @@
+        <div class="row label-pair ml-1 mb-2">
+          <div id="permitNumber"><b>Mines Act Permit Number: </b>{{mine?.permitNumbers && mine?.permitNumbers.length > 0 ? mine?.permitNumbers?.join() : '-'}}</div>
+        </div>
+        <div class="row label-pair ml-1 mb-2">
+          <div id="tailingsImpoundments"><b>Number of Tailings Storage Facilities: </b>{{mine?.tailingsImpoundments || '-' }} </div>
+        </div>
+        <div class="row label-pair ml-1 mb-2">
+          <div id="commodities"><b>Commodities: </b>{{mine?.commodities && mine?.commodities?.length > 0 ? mine?.commodities?.join(', ') : '-'}}</div>
+        </div>
+      </div>
+      <!-- column 3/3 -->
+      <div class="col-lg-4 col-md-12">
+        <div class="row label-pair ml-1 mb-2">
+          <div id="region"><b>Region: </b>{{mine?.region || '-' }}</div>
+        </div>
+        <div class="row label-pair ml-1 mb-2">
+          <div id="longitude"><b>Longitude: </b>{{mine?.location ? (mine?.location.coordinates[0]) : '-'}}</div>
+        </div>
+        <div class="row label-pair ml-1 mb-2">
+          <div id="latitude"><b>Latitude: </b>{{mine?.location ? (mine?.location.coordinates[1]) : '-'}}</div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr>
+
   <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Name:</label>
-    <div>{{record?.name || '-'}}</div>
-  </div>
-  <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Permit Numbers:</label>
-    <div>{{record?.permitNumbers && record?.permitNumbers.length > 0 ? record?.permitNumbers?.join() : '-'}}</div>
-  </div>
-  <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Status:</label>
-    <div>{{record?.status || '-'}}</div>
-  </div>
-  <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Type:</label>
-    <div>{{record?.type || '-'}}</div>
-  </div>
-  <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Commodities:</label>
-    <div>{{record?.commodities && record?.commodities?.length > 0 ? record?.commodities?.join() : '-'}}</div>
-  </div>
-  <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Tailings Impoundments:</label>
-    <div>{{record?.tailingsImpoundments || '-' }}</div>
-  </div>
-  <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Region:</label>
-    <div>{{record?.region || '-' }}</div>
-  </div>
-  <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Permittee:</label>
-    <div>{{record?.permittee || '-' }}</div>
+    <label class="font-weight-bold mr-1">Mine Type:</label>
+    <div>{{mine?.type || '-'}}</div>
   </div>
   <div class="row label-pair ml-1 mb-2">
-    <label class="font-weight-bold mr-1">Summary:</label>
-    <div>{{record?.summary || '-' }}</div>
+    <label class="font-weight-bold mr-1">Mine Description:</label>
```

Comment:
When mine description/summary are longer, they'll wrap onto a second line. Which is good. It looks strange though when one is wrapped and the other is inline.

Maybe Have the label always display above the text? Just put the text on it's own row and the label above.